### DISCUSSION TITLE
Step 1, Create a branch: All jobs have failed

### DISCUSSION THREAD

**Discussion by h4rz** [skills/.github#161] [Q&A]
I'm doing the `Introduction to GitHub` from the GitHub skills (https://github.com/skills/introduction-to-github) and receiving the below error message. Can someone please help me?

<img width="1439" alt="image" src="https://user-images.githubusercontent.com/20818641/216801730-7a95c487-4362-47bf-8c88-8913831e38a3.png">

**Comment by SlimBloodworth** [comment]
It's saying that the HEAD of your branch is behind the remote branch. You need to do a pull request from the remote repo before you can push so that the HEAD branch from your current branch and the HEADF from the remote branch have the same info and are integrated so that they are on the same HEAD.
try: git pull <the-remote-name-here> <the-branch-name-here>

**Comment by sinsukehlab** [comment]
This error occurred because workflows are not permitted to create pull requests. The restriction can probably be circumvented by running shell scripts instead of commands.

**Reply by sinsukehlab** [reply]
See: https://github.blog/changelog/2023-02-02-github-actions-updating-the-default-github_token-permissions-to-read-only/

**Reply by sinsukehlab** [reply]
Oh, I have mistaken.
The introduction-to-github course doesn't automatically create pull requests.

**Comment by nrogoff** [comment]
You can change the permissions in the repo's settings. Delete the new branch and then re-run the 'Step 0, Start' action. That gets things back on track.

![image](https://user-images.githubusercontent.com/1227639/218308964-634a6346-57ff-4601-a261-d8286bdea78e.png)

**Comment by sinsukehlab** [comment]
This error occurred because the env `$BASE_BRANCH_NAME` was not set in the action skills/action-update-step@v1.2.0 used in the workflows in the course.
Now that skills/action-update-step@v1.2.1 is released, the error is fixed.
